packages/extension-ui/src/Popup/AuthManagement/index.tsx

```diff
@@ -74,8 +89,9 @@ function AuthManagement({ className }: Props): React.ReactElement<Props> {
       <div className={className}>
         {!authList || !hasAuthList ? (
           <div className='empty-list'>
-            <Svg
+            <img
               className='animated-trusted'
+              key={Math.floor(Math.random() * Date.now())}
               src={animTrusted}
             />
```

Comment:
If you need that in few places probably a small new image component that has timestamp as key would be a good option